sale_pricelist_global_rule/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,3 @@
+* `Tecnativa <https://www.tecnativa.com>`_
+
+  * Carlos López
```

Comment:
You should include me as the designer (and now reviewer).

Reply:
done